src/main/java/io/codelex/flightplanner/api/FlightService.java

```diff
@@ -0,0 +1,45 @@
+package io.codelex.flightplanner.api;
+
+import io.codelex.flightplanner.classes.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Service
+public class FlightService {
```

Comment:
Liels pluss par to, ka service ir tik tīrs un, ka faktiski visu dara repository - tas atvieglos dzīvi, kad pāriesim uz datu bāzi.

Reply:
Tas labi, jo es jau beigās nedaudz sāku šaubīties vai pareizi ka viss tikai repository.. 😄